### PR TITLE
Add the YAML file for edge nodes to the Helm file

### PR DESCRIPTION
**Pull request by wenchajun**
### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: charts/fluent-operator/values.yaml [hunk 1]
```diff
@@ -180,6 +180,12 @@ fluentbit:
     systemd:
       enable: true
 
+  edge:
```
Comment (benjaminhuo) [inline charts/fluent-operator/values.yaml:183]: edge can be changed directly to `kubeedge` to be more specific

File: charts/fluent-operator/templates/output-stdout-edge.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{{- if .Values.fluentbit.edge.enable -}}
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterOutput
```
Comment (benjaminhuo) [inline charts/fluent-operator/templates/output-stdout-edge.yaml:3]: stdout output for edge can be removed

Comment (benjaminhuo) [inline charts/fluent-operator/templates/output-stdout-edge.yaml:3]: stdout output for edge can be removed

File: charts/fluent-operator/values.yaml [hunk 3]
```diff
@@ -180,6 +180,12 @@ fluentbit:
...
+    enable: false
+    prometheusRemoteWrite:
+      host: "<cloud-prometheus-service-host>"
```
Comment (benjaminhuo) [inline charts/fluent-operator/values.yaml:186]:
```suggestion
      # change the host to a cloud-side Prometheus-compatible server that can receive Prometheus remote write data
      host: "<cloud-prometheus-service-host>"
```

File: charts/fluent-operator/values.yaml [hunk 4]
```diff
@@ -180,6 +180,12 @@ fluentbit:
...
+      port: "<cloud-prometheus-service-port>"
```
Comment (benjaminhuo) [inline charts/fluent-operator/values.yaml:187]:
```suggestion
     ```suggestion
      # change the port to the port of a cloud side Prometheus-compatible server that can receive Prometheus remote write data
      port: "<cloud-prometheus-service-port>"
```

**Comment by wenchajun** [comment]
fixed

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**